Makefile

```diff
@@ -21,3 +21,7 @@ clean:
 
 app:
 	@$(REBAR) -r create template=mochiwebapp dest=$(DEST) appid=$(PROJECT)
+
+bench: app
```

Comment:
Perhaps little bit late, but would it be more *Makefile*-ish to do:

```make
.PHONY: bench
bench: app $(DEST)/$(PROJECT).sh

$(DEST)/$(PROJECT).sh:
	@cp support/bench.sh $(DEST)/bench.sh
``` 
One other question, why does it says "Writing $(DEST)/**$(PROJECT)**.sh" but actually creating "$(DEST)/**bench**.sh"?